### DISCUSSION TITLE
Is there a way to make text glow?

### DISCUSSION THREAD

**Discussion by stevenliebregt** [slint-ui/slint#3978] [Q&A] [answered]
Hi I'm looking around for something to write a UI with for a project and would like to make text appear to be glowing like in the following example is possibly with for example CSS: [https://www.w3schools.com/howto/howto_css_glowing_text.asp](https://www.w3schools.com/howto/howto_css_glowing_text.asp). Is something like this possible with Slint? I tried looking through some examples and documentation but could not figure something out.

**Comment by hunger** [comment, marked as answer]
We are still missing text formatting (#2723). I guess this goes even beyond normal formatting though.

Slint has a [`DropShadow`](https://slint.dev/releases/1.3.0/docs/slint/src/language/builtins/elements#drop-shadows) builtin, but that works with `Rectangle`s only at this time.

So Slint can not do that right now with the built-in tools, but maybe you can embed an SVG? That should be capable of any effect;-)

**Reply by stevenliebregt** [reply]
Did not think about using SVG's, that could indeed work, thanks
